package.json

```diff
@@ -24,6 +24,8 @@
     "lint:fix": "npm run lint:eslint:fix",
     "lint:eslint": "eslint \"./**/*.js\"",
     "lint:eslint:fix": "eslint \"./**/*.js\" --fix",
+    "prettier-check": "prettier --check \"**/*.js\"",
```

Comment:
For consistency with the other npm scripts, can we rename these:
`prettier-check` -> `prettier`
`prettier-fix` -> `prettier:fix`